execCommand.html

```diff
@@ -795,8 +772,7 @@ <h2 id="methods-to-query-and-execute-commands">Methods to query and
                         <a class="external" data-anolis-spec="dom" href=
                         "http://dom.spec.whatwg.org/#concept-event" title=
                         "concept-event">event</a> at <var title="">affected
```

Comment:
```suggestion
                        "concept-event">event</a> at <var>affected
```

Reply:
Could you restore the preceding space?

Reply:
`var title=""` is everywhere as `data-anolis-spec` is. Maybe in another cleanup?

Reply:
yeah, we can get rid of all of the `data-anolis-spec` stuff... separate PR?

Reply:
It seems the spec needs some serious cleanup as there are so many obsolete things, like mentioning Firefox 6.0a1 😄

Reply:
lol, yeah :)... it needs a little love. Feel free to send patches.